### DISCUSSION TITLE
Accessibilité

### DISCUSSION THREAD

**Discussion by IDADV** [microsoft/vscode-discussions#2463] [Extension Development QnA] [answered]
bonjour je me présente je m'appelle Youssef et je suis malvoyant je me permets de vous contacter car j'ai un projet qui est de développer et adapter les logiciels qui existent de nos jours pour déficience visuelle pour que eux développent eux aussi leur imagination et des projets qui et c'est pour cela que je me permets de vous contacter car j'aimerais qu'ensemble nous travaillons pour que visual studio code soit accessible pour déficient visuelle merci d'avoir lu ce message en attente de votre réponse merci beaucoup et bonne soirée 

bien cordialement 
Youssef

**Comment by alefragnani** [comment, marked as answer]
Hi @IDADV ,

Visual Studio Code has many features to help make the editor accessible to all users. Zoom levels and High Contrast colors improve editor visibility, keyboard-only navigation supports use without a mouse, and the editor is optimized for screen readers. Also, improvements in accessibility are released every month, on each new Visual Studio Code release. You can read more detailed info at https://code.visualstudio.com/docs/editor/accessibility.

But, if you have issues or suggestions while using Visual Studio Code, please report in the Visual Studio Code repo, at https://github.com/microsoft/vscode. This discussions forum is for extension development only for now.

Hope this helps
